configure

```diff
@@ -576,7 +576,8 @@ def get_version_helper(cc, regexp):
 
 def get_llvm_version(cc):
   return get_version_helper(
-      cc, r"(^clang version|based on LLVM) ([3-9]\.[0-9]+)")
+      cc, r"(^clang version|^FreeBSD clang version|based on LLVM) " +
+      "([3-9]\.[0-9]+)")
```

Comment:
I'm a little coward ;-). 
`get_xcode_version` in 4 lines below had also 4 space indents. I fixed both. Thanks.